src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableClient.cs

```diff
@@ -136,6 +137,19 @@ async Task<string> IDurableOrchestrationClient.StartNewAsync(string orchestrator
             return instance.InstanceId;
         }
 
+        private bool TryGetStatusesNotToOverride(out OrchestrationStatus[] dedupeStatuses)
+        {
+            var overridableStates = this.config.Options.OverridableExistingInstanceStates;
+            if (overridableStates == OverridableStates.IfTerminatedFailedOrCompleted)
+            {
+                dedupeStatuses = new OrchestrationStatus[] { OrchestrationStatus.Running, OrchestrationStatus.ContinuedAsNew, OrchestrationStatus.Canceled, OrchestrationStatus.Pending };
```

Comment:
I suggest formatting this so that each entry is on its own line. It makes reading the code a bit easier (and comparing diffs). Like this:

```csharp
dedupeStatuses = new OrchestrationStatus[]
{
    OrchestrationStatus.Running,
    OrchestrationStatus.ContinuedAsNew,
    OrchestrationStatus.Pending,
};
```

Also, I don't think we need the `Canceled` status check here (DTFx doesn't actually use this status for anything).